webscraping/node/course-info/storer.js

```diff
@@ -10,44 +10,44 @@
 /**
  * imports
  */
-var fs = require('fs');
-var assert = require('assert');
-var MongoClient = require('mongodb').MongoClient;
-var argv = require('minimist')(process.argv.slice(2));
+let fs = require('fs');
+let assert = require('assert');
+let MongoClient = require('mongodb').MongoClient;
+let argv = require('minimist')(process.argv.slice(2));
 
 /**
  * constants
  */
 const DIR = '../../r/course-info-jsonfiles/';
 const jsonFilenameRegex = /_document.json/;//filter by filename
-var mongoServerUrl = 'mongodb://138.197.6.26:27017/';
-var devDbName = "courseplannerdb-dev";
-var prodDbName = "courseplannerdb";
-var dbName = (argv.prod) ? prodDbName : devDbName;
-var dbFullUrl = mongoServerUrl + dbName;
+let mongoServerUrl = 'mongodb://138.197.6.26:27017/';
+let devDbName = "courseplannerdb-dev";
+let prodDbName = "courseplannerdb";
+let dbName = (argv.prod) ? prodDbName : devDbName;
+let dbFullUrl = mongoServerUrl + dbName;
 
```

Comment:
The dev DB is useful for testing changes to our data in it's interaction with the other components of our application. We had never discussed transferring the contents of the dev DB into the prod DB. In fact, the way we have it set up means that only the cron job will ever run this file in 'prod mode'. The whole point of the cron job is to avoid manual updating of the production database. I hope this answers your question as I'm surprised by it.